src/server/game/Movement/Spline/MovementPacketBuilder.cpp

```diff
@@ -98,7 +98,7 @@ namespace Movement
 
     void WriteLinearPath(const Spline<int32>& spline, ByteBuffer& data)
     {
-        uint32 last_idx = spline.getPointCount() - 3;
+        uint32 last_idx = spline.getPointCount() - 2;
```

Comment:
The old code is correct, client magic, do not touch. These ending points are used serverside for catmullrom interpolation only and should not be sent